scripts/travis/codegen_verification.sh

```diff
@@ -56,7 +56,7 @@ function runGoLint() {
 }
 
 echo "Running go vet..."
-go vet $(go list ./... | grep -v /test/e2e-go/)
```

Comment:
Consider `make vet` instead

Reply:
@winder Sorry, I should have considered reusing an existing `make` target.  Thanks for calling it out - addressed via https://github.com/algorand/go-algorand/pull/4312/commits/f6603bdcf2ed25e093d37b239bcb6e945a62187c.

Resolving to merge the PR.